providers/src/airflow/providers/microsoft/azure/hooks/adx.py

```diff
@@ -20,7 +20,7 @@
 
 .. spelling:word-list::
 
-    KustoResponseDataSetV
+    KustoResponseDataSet
```

Comment:
Ah, cool did not notice and did not know that spelling wordlists can also be adjusted non-global in a py file locally...